.github/workflows/publish-js-libs.yml

```diff
@@ -31,11 +26,11 @@ jobs:
       - name: Set up Node
         uses: actions/setup-node@v1
         with:
-          node-version: '12.x'
-          registry-url: 'https://registry.npmjs.org'
+          node-version: "12.x"
+          registry-url: "https://registry.npmjs.org"
       - name: Generate JS libs
         run: |
           make js-protos
       - name: Publish JS libs
         run: |
-          ./scripts/publish_js_protos.bash -v ${{ steps.latesttag.outputs.tag }} -t ${{ secrets.NPM_AUTH_TOKEN }}
```

Comment:
I don't see why we need to use that 3rd party action to get the tag. Since this runs with a release trigger, we can just get the tag directly from the context as well as whether or not this is a prerelease.

Reply:
I think it was just so that it's testable outside of a release trigger, but SGTM.